Docs 7741 clean up deploying to cloudhub hn (#344)
mulesoft/docs-runtime-manager: modules/ROOT/pages/cloudhub-manage-props.adoc

.The arrow shows the variable on the *Properties* tab of the application *Settings* page.
image::rtm-env-variables.png[*Properties* tab on the *Settings* page]

+
[NOTE]

> In the *Text* view, use the backslash (`\`) character as an escape character.

Reviewer: ```suggestion
In the *Text* view, use the backslash character (`\`) as an escape character.
```

> For example, to use `:` in key or value, use `\:` in the text.

Reviewer: ```suggestion
For example, to use `:` in a key or value, use `\:` in the syntax.
```

> To use `\` in a key or value, use `\\` in the text.

Reviewer: ```suggestion
To use `\` in a key or value, use `\\` in the syntax.
```

> No escape character is required in list format.

Reviewer 2: in list format -> in *List* view? feel it would be better to keep it consistent with *Text* view

> No escape character is required in the *List* view.

Reviewer: ```suggestion
*List* view syntax requires no escape character.
```
